src/NumFieldOrd/NfOrd/Clgp/Saturate.jl

```diff
@@ -519,10 +522,14 @@ function simplify(c::Hecke.ClassGrpCtx, U::Hecke.UnitGrpCtx, cp::Int = 0; use_LL
   new_rels = Vector{FacElem{AbsSimpleNumFieldElem, AbsSimpleNumField}}()
   vals_new_rels = Vector{SRow{ZZRingElem}}()
   @vprintln :Saturate 1 "Computing rels..."
+  scan = true
+  #basis is in HNF, the saturation only needs the "essential" bit,
+  #that is all starting at the first none-one entry
```

Comment:
no longer convinced this is neccessary, this is not wrong, but too much work. Reasoning: if the matrix is, locally, (p 0 *; 0 1 *) then th eonly possibility of getting a p in 2nd position is to add p x the 2nd, but if this is then a p-th power, the 1st row alone would have already been...